### ISSUE TITLE
googlenet eval mode

### ISSUE THREAD

**Issue by pcshih** [weirme/FCSN#16]
https://github.com/weirme/Video_Summary_using_FCSN/blob/0895cccbb2a488369b1bfc7d2c087b3050250898/make_dataset.py#L53-L56

Because googlenet is only for feature extraction, it should be in eval mode.

[Remember that you must call model.eval() to set dropout and batch normalization layers to evaluation mode before running inference. Failing to do this will yield inconsistent inference results.](https://pytorch.org/tutorials/beginner/saving_loading_models.html?highlight=eval)
